src/test/java/org/jitsi/meet/test/pageobjects/web/WebRemoteParticipant.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright @ 2018 8x8 Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.pageobjects.web;
+
+import org.jitsi.meet.test.pageobjects.base.*;
+import org.jitsi.meet.test.util.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.*;
+
+/**
+ * The web implementation of <tt>RemoteParticipant/tt>.
+ *
+ * @author Damian Minkov
+ */
+public class WebRemoteParticipant
+    extends RemoteParticipant<WebDriver>
+{
+    /**
+     * Constructs <tt>WebRemoteParticipant</tt>.
+     * @param driver the driver.
+     * @param endpointId the endpoint id.
+     */
+    public WebRemoteParticipant(WebDriver driver, String endpointId)
+    {
+        super(driver, endpointId);
+    }
+
+    @Override
+    public void mute()
+    {
+        clickOnRemoteMenuLink("mutelink");
+    }
+
+    @Override
+    public void kick()
+    {
+        clickOnRemoteMenuLink("kicklink");
+    }
+
+    /**
+     * Opens the remote menu and waits for the desired link to appear.
+     * @param linkClassname the class name to use identifying the link.
+     */
+    private void clickOnRemoteMenuLink(String linkClassname)
+    {
+        // Open the remote video menu
+        WebElement cntElem
```

Comment:
What is cnt? I'd prefer this be named to what it is, without abbreviation, because I don't think cnt is a common abbreviation.

Reply:
Haha, no idea :) I just moved the code. I will change it with something more readable.